src/mito-auth.lisp

```diff
@@ -53,7 +53,13 @@
   (when password
     (setf (password object) password)))
 
+(defun normalize-password-salt (password-salt)
+  (let ((trimmed-password-salt (str:trim password-salt)))
+    (if (str:starts-with-p "\\x" trimmed-password-salt)
```

Comment:
Can these be rewritten without `str`, perhaps with `cl:string-trim` and `uiop:string-prefix-p`?